packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -93,14 +94,30 @@ export class MirrorNodeClient {
     protected createAxiosClient(
         baseUrl: string
     ): AxiosInstance {
-        return Axios.create({
+        const axiosClient: AxiosInstance = Axios.create({
             baseURL: baseUrl,
             responseType: 'json' as const,
             headers: {
                 'Content-Type': 'application/json'
             },
             timeout: 10 * 1000
         });
+
+        // @ts-ignore
+        axiosRetry(axiosClient, {
+            retries: 10,
+            retryDelay: (retryCount) => {
+                this.logger.info(`Retry delay ${retryCount * 1000} s`);
+                return retryCount * 1000;
+            },
+            retryCondition: (error) => {
+                this.logger.error(error, `Request failed`);
+                return error?.response?.status === 400 || error?.response?.status === 404;
```

Comment:
Retry condition should take into account allowable errors.
Can you see if it's possible to utilize the `allowedErrorStatuses` list to determine?

If not then let's note this and open a separate ticket to enhance this in the future.

Reply:
https://github.com/hashgraph/hedera-json-rpc-relay/issues/277